### PR TITLE
Bugfix: Equivalence issues

### PR DESCRIPTION
**Pull request by joeflack4**
Addresses:
- Equivalence problem mentioned here: https://github.com/monarch-initiative/mondo-ingest/pull/412#discussion_r1493469285

## Overview
This PR is a workaround for illogical class equivalence issues highlighted by the ontological reasoner. After applying this fix, it is expected that no such error will be thrown when running `build-mondo-ingest`.

## Pre-merge checklist
- [x] Docs
   - `docs/` have been added/updated **OR**
   - No updates to the docs necessary after careful consideration.
- [x] QC
   - `sh run.sh make build-mondo-ingest` has been run on this branch (after `docker pull obolibrary/odkfull:dev), and no errors occurred **OR**
   -  No functional (code-related) changes to the pipeline are suggested, so no re-run is necessary.
- [x] Account for any new packages
  - n/a
- [x] Reviewed
   - Has been sufficiently reviewed by at least one review from a different team member of the Mondo Technical team.

## Changes
Bugfix: Equivalence issues
Temporary fix for: `ERROR org.obolibrary.robot.ReasonOperation - Only equivalent classes that have been asserted are allowed. Inferred equivalencies are forbidden.`
- Update: makefile: Goal for removing equivalents axioms. See https://github.com/monarch-initiative/mondo-ingest/pull/412#discussion_r1494224976

### CODE REVIEW

**Comment by twhetzel** [comment]
This looks ok.

File: makefile [hunk 1]
```diff
@@ -8,8 +8,14 @@ SOURCE_URL=https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
 # MAIN COMMANDS / GOALS ------------------------------------------------------------------------------------------------
 all: tmp/output/release/icd11foundation.owl
 
+clean:
+	rm -rf tmp/
+
+tmp/output/release/icd11foundation.owl: tmp/output/intermediate1-unicode-cleaned.owl
+	robot remove --axioms equivalent -i $< -o $@
```
Comment (joeflack4) [inline makefile:15]: ## The essential change
This is the command that applies the fix.

- [x] @joeflack4 Run `build-mondo-ingest` using the output from this to verify if it worked.

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
```
Comment (joeflack4) [inline run.sh:1]: ## `run.sh`
This is a dependency / proxy for dependencies. Adding this file and using it is necessary to ensure the the ODK container is running, with all the correct versions of everything (e.g. `robot` in this case).

@twhetzel I don't like that adding this file is how I handle this dependency. The only other repo that currently does this is MedGen, and the way that I update it is to copy the `run.sh` manually from `mondo-ingest`. I did bring this up to Nico once prior, that I would prefer a solution such as a Python package that puts a wrapper around `run.sh`. Not a high priority I think, but would be nice eventually.

<details><summary>Related</summary>
<p>

- https://github.com/monarch-initiative/mondo-ingest/issues/446
- https://github.com/monarch-initiative/mondo-ingest/pull/447
- https://github.com/monarch-initiative/medgen/pull/17

</p>
</details>

File: README.md [hunk 3]
```diff
@@ -1,2 +1,11 @@
...
+
+## Pre-reqs
+- Docker
+- Docker images  
+  One or both of the following, depending on if you want to run the stable build `latest` or `dev`:
+  - a. `docker pull obolibrary/odkfull:latest`
+  - b. `docker pull obolibrary/odkfull:dev`
+
+## Running
+`sh run.sh make all`
```
Comment (joeflack4) [inline README.md:11]: ## Docs
Now I'm bundling other things that could be put into another PR. But after adding `run.sh`, I realized I needed docs for it, so I updated the readme with new sections:
- Pre- reqs
- Running

File: makefile (hunk 1 above)
Comment (twhetzel) [reply makefile:15]: I agree, I think that line (`robot remove --axioms equivalent -i $< -o $@`) is fine. 

I'm not sure if https://robot.obolibrary.org/remove#preserving-the-structure is also needed. I think that was needed with something in the Mondo repo in Sep 2023, but I don't recall the details or am able to find it in the repo files yet. I am leaving this note here as a reminder in case things do not look as expected.

File: README.md (hunk 3 above)
Comment (twhetzel) [reply README.md:11]: Thanks, looks good.

File: makefile [hunk 4]
```diff
@@ -8,8 +8,14 @@ SOURCE_URL=https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
 # MAIN COMMANDS / GOALS ------------------------------------------------------------------------------------------------
 all: tmp/output/release/icd11foundation.owl
 
+clean:
```
Comment (twhetzel) [inline makefile:11]: Is `clean` still being used?

File: run.sh (hunk 2 above)
Comment (twhetzel) [reply run.sh:1]: Hmm, at the moment I'm not sure what the best solution/idea would be.... 
- [x] 1. Add ticket (low priority) to track this possible future work
- [x] 2. Is this step documented in the main README for mondo ingest?

File: makefile (hunk 4 above)
Comment (joeflack4) [reply makefile:11]: I just added that. I added it just as a command for convenience.

File: makefile (hunk 1 above)
Comment (joeflack4) [reply makefile:15]: I will look at your comment tomorrow, but I just wanted to report back here that [the build passed](https://github.com/monarch-initiative/mondo-ingest/pull/412#discussion_r1496830887)!

_edit_: I looked now at what you wrote yesterday; I see that this is a note for the future if needed. FYI I looked at those docs and it does look like by default the structure is preserved by default (to not preserve, you'd do ` --preserve-structure false`, so that's good. I wonder if in Mondo they had been explicitly not preserving the structure and then later found that to be a mistake.

File: makefile (hunk 4 above)
Comment (twhetzel) [reply makefile:11]: Is it still needed now though?

File: run.sh (hunk 2 above)
Comment (joeflack4) [reply run.sh:1]: I may have already done (1) but it's on my todo list to check and do so shortly. Regarding (2), no and also on my todo list; may have a PR for that today!

_edit:_
(1) is done!
- https://github.com/monarch-initiative/mondo-ingest/issues/446

(2) done via:
- https://github.com/monarch-initiative/mondo-ingest/pull/447

File: makefile (hunk 4 above)
Comment (joeflack4) [reply makefile:11]: It wasn't ever strictly needed. It's just there for convenience, and I don't think it's a harm to have such utilities.. I've seen this convention in a variety of other projects, such as [in MedGen](https://github.com/monarch-initiative/medgen/blob/a5d64231c9c233cedddd9e9f01a1d21c2992e255/makefile#L29); I committed this, but this makefile goal (and much of the MedGen makefile) was copied over from a different location from work done by Chris Mungall.
